test/dynamo/test_misc.py

```diff
@@ -3979,6 +3979,22 @@ def foo(mod, x):
         self.assertIsNone(mod_ref(), None)
         self.assertIsNone(mod_weight_ref(), None)
 
+    def test_release_scope_memory(self):
+        def inner(y):
+            y
+
+        inner = torch._dynamo.optimize("eager")(inner)
+
+        p_ref = None
+
+        x = torch.randn((10, 10), device="cuda:0")
```

Comment:
```suggestion
        x = torch.randn((10, 10), device="cuda")
```
nit i don’t think we should specify these whenever possible for tests